corretto8/plan.sh

```diff
@@ -22,18 +22,26 @@ do_build() {
 
 do_install() {
   pushd "${pkg_prefix}" || exit 1
-  rsync -avz "$HAB_CACHE_SRC_PATH/${pkg_dirname}/" .
+  rsync -avz "${HAB_CACHE_SRC_PATH}/${pkg_dirname}/" .
 
-  export LD_RUN_PATH="${LD_RUN_PATH}:${pkg_prefix}/lib/amd64/jli:${pkg_prefix}/lib/server:${pkg_prefix}/lib/amd64"
+  LD_RUN_PATH="${LD_RUN_PATH}:${pkg_prefix}/lib/amd64/jli"
+  LD_RUN_PATH="${LD_RUN_PATH}:${pkg_prefix}/lib/server"
+  LD_RUN_PATH="${LD_RUN_PATH}:${pkg_prefix}/lib/amd64"
+  LD_RUN_PATH="${LD_RUN_PATH}:${pkg_prefix}/jre/lib/amd64"
+  LD_RUN_PATH="${LD_RUN_PATH}:${pkg_prefix}/jre/lib/amd64/server"
+  export LD_RUN_PATH
 
   build_line "Setting interpreter for all executables to '$(pkg_path_for glibc)/lib/ld-linux-x86-64.so.2'"
-  build_line "Setting rpath for all libraries to '$LD_RUN_PATH'"
+  build_line "Setting rpath for all libraries to '${LD_RUN_PATH}'"
 
-  find "$pkg_prefix"/bin -type f -executable \
+  find "${pkg_prefix}"/bin -type f -executable \
     -exec sh -c 'file -i "$1" | grep -q "x-executable; charset=binary"' _ {} \; \
     -exec patchelf --interpreter "$(pkg_path_for glibc)/lib/ld-linux-x86-64.so.2" --set-rpath "${LD_RUN_PATH}" {} \;
 
-  find "$pkg_prefix/lib" -type f -name "*.so" \
+  find "${pkg_prefix}/lib" -type f -name "*.so" \
+    -exec patchelf --set-rpath "${LD_RUN_PATH}" {} \;
+
+  find "${pkg_prefix}/jre/lib" -type f -name "*.so" \
```

Comment:
If I remember correctly, this path was missing in corretto8, but present in corretto11. Is this now present here?

Reply:
@johnjelinek Did you have issues with the build?

The output above shows the paths that are linked. Including:

```
libjava.so => /hab/pkgs/rakops/corretto8/8.202.08.2/20190612021235/jre/lib/amd64/libjava.so (0x00007f63a5334000)
libjvm.so => /hab/pkgs/rakops/corretto8/8.202.08.2/20190612021235/jre/lib/amd64/server/libjvm.so (0x00007f63a43b7000)
```